README.md

```diff
@@ -30,7 +30,7 @@ All OSs currently supported by snap:
 ### Installation
 
 #### Download the plugin binary:
-You can get the pre-built binaries for your OS and architecture at snap's [GitHub Releases](https://github.com/intelsdi-x/snap/releases) page. Download the plugins package from the latest release, unzip and store in a path you want `snapd` to access.
+You can get the pre-built binaries for your OS and architecture at plugins's [GitHub Releases](https://github.com/intelsdi-x/snap-plugin-collector-smart/releases) page. Download the plugins package from the latest release, unzip and store in a path you want `snaptel` to access.
```

Comment:
Please correct the link to release page, now the link indicates releases page for smart collector plugin instead of releases page for users collector plugin.